bentoml/utils/__init__.py

```diff
@@ -261,3 +265,19 @@ def resolve_bento_bundle_uri(bento_pb):
         return bento_pb.uri.gcs_presigned_url
     else:
         return bento_pb.uri.uri
+
+
+def archive_directory_to_tar(source_dir, tarfile_dir, tarfile_name):
+    file_name = f'{tarfile_name}.tar'
+    tarfile_path = os.path.join(tarfile_dir, file_name)
+    with tarfile.open(tarfile_path, mode="w:gz") as tar:
+        # Use arcname to prevent storing the full path to the bundle,
+        # from source_dir/path/to/file to path/to/file
+        tar.add(source_dir, arcname='')
```

Comment:
can we use `arcname="bento_name"` so it's consistent with how we currently create zip and upload bundle to s3? https://github.com/bentoml/BentoML/blob/master/bentoml/saved_bundle/bundler.py#L60